### DISCUSSION TITLE
Role in NHCX

### DISCUSSION THREAD

**Discussion by vishi24** [NHA-ABDM/nhcx#8] [Q&A] [answered]
How should I determine which role to assign myself in the NHCX system?

**Comment by akhilsiddharth** [comment, marked as answer]
There are two primary roles in NHCX - Provider or Payer.
An insurer is a Payer on NHCX, a solution catering to the provider side ( hospital, lab, pharmacy, etc ) is a Provider on NHCX.

Based on where your software/solution will be deployed, the role will accordingly be determined.
